prov/util/src/util_attr.c

```diff
@@ -134,6 +134,15 @@ static int ofix_alter_layer_info(const struct fi_provider *prov,
 	if (!(*base_info = fi_allocinfo()))
 		return -FI_ENOMEM;
 
+	if (layer_info && layer_info->fabric_attr) {
+		if (layer_info->fabric_attr->name) {
+			(*base_info)->fabric_attr->name = strdup(layer_info->fabric_attr->name);
+			if (!(*base_info)->fabric_attr->name)
+				goto err;
+		}
+		(*base_info)->fabric_attr->api_version = layer_info->fabric_attr->api_version;
+	}
```

Comment:
Okay. I'll drop these changes. Setting the core api_version could be left to the individual util providers.